tests/test_tabulate.py

```diff
@@ -112,11 +118,43 @@ def test_Fe_user(check_tabulate,system_Fe_W90, compare_fermisurfer,quantities_ta
 def test_Chiral(check_tabulate,system_Chiral, compare_fermisurfer,quantities_tab):
     """Test Energies, Velocities, berry curvature, its derivative"""
     check_tabulate(system_Chiral , quantities_tab , frmsf_name="tabulate_Chiral" , suffix="" ,  comparer=compare_fermisurfer,
-              additional_parameters = {'external_terms':False}, ibands = [0,1] )
+              parameters = {'external_terms':False}, ibands = [0,1] )
 
 
 def test_Chiral_sym(check_tabulate,system_Chiral, compare_fermisurfer,quantities_tab):
     """Test Energies, Velocities, berry curvature, its derivative"""
     check_tabulate(system_Chiral , quantities_tab , frmsf_name="tabulate_Chiral" , suffix="sym" ,  comparer=compare_fermisurfer,
-               use_symmetry =  True  , additional_parameters = {'external_terms':False}, ibands = [0,1] )
-
+               use_symmetry =  True  , parameters = {'external_terms':False}, ibands = [0,1] )
+
+
+def test_CuMnAs_PT(check_tabulate,system_CuMnAs_2d_broken,compare_fermisurfer):
+    """here no additional data is needed, we just check that degen_thresh=0.05 and degen_Kramers=True give the same result"""
```

Comment:
Could you mention in a comment that this tests the tabulation of user-defined quantity?

Also, you seem to add a reference result `tabulate_CuMnAs_omega2-zz.frmsf`, so I am confused what you mean by "here no additional data is needed".

Reply:
> Could you mention in a comment that this tests the tabulation of user-defined quantity?

done

> Also, you seem to add a reference result tabulate_CuMnAs_omega2-zz.frmsf, so I am confused what you mean by "here no additional data is needed".

yes, it was a copypaste from integrate, here I actually add a reference data. I corrected,

Thank you.